JS-108_javascript-basics/code-challenge/create-a-variable/app.js

```diff
@@ -0,0 +1,2 @@
+// Create a variable named 'myName'
+var myName = "Paul Eliczar Adraneda";
```

Comment:
It's not advisable anymore to use `var` unless there's a need to. In this case, change `var` to `const`.